### ISSUE TITLE
Strange behaviour when panes defined as multi-element dictionaries

### ISSUE THREAD

**Issue by podung** [tmuxinator/tmuxinator#406]
There appears to be a bug in the handling of a pane element that contains a multi-element dictionary.  I found it when working on a PR for #266.

```yaml
# This works: A pane with 2 elements in an array, each being a single-element dictionary
panes:
  - pane1: echo "first pane"
  - pane2: echo "second pane"
# results in the expected 2 panes, each running its respective echo command.
```

```yaml
# This doesn't: An array that contains a multi-element dictionary
panes:
  - pane1: echo "first pane"
    pane2: echo "second pane"
# bizarrely creates three panes
# All the commands run in the first pane, the next two panes are empty panes
```

The options I see are:
1. Take the first element and ignore the rest (this is consistent with how multi-element Windows yaml is handled (`value = window_yaml.values.first` in window.rb:18)
2. Make each element its own pane
3. Print out some sort of invalid format error

I know its a corner case that's pretty minor, but I'm trying clean up the surrounding code a bit while I address #266.

Any thoughts?  I'm open to any suggestions.

**Comment by podung** [comment]
For anyone interested, the source of this bug is the pane_yml hash gets mapped and a new Pane is created for each entry.  The problem is the same index is passed to each of these objects:

```ruby
# window.rb

def build_panes(panes_yml)
  Array(panes_yml).map.with_index do |pane_yml, index|
    if pane_yml.is_a?(Hash)
      pane_yml.map do |_name, commands|
        Tmuxinator::Pane.new(index, project, self, *commands)
      end
    else
      Tmuxinator::Pane.new(index, project, self, pane_yml)
    end
  end.flatten
end
```

In the sample case above this creates 2 Pane objects each with an index of 0.  When the template.erb outputs the pane commands all of the commands target get run against pane 0.  

Additionally this breaks the logic for determining `pane.last?`, resulting in 1 pane that runs all the commands and two additional empty panes.

**Comment by podung** [comment]
I guess each hash element could also be interpreted as a command to run in the pane.  But its kind of weird to name your commands with hash keys.

**Comment by podung** [comment]
Closed by #418
